parallel/utils.py

```diff
@@ -0,0 +1,9 @@
+import os
+from glob import glob
+
+
+def get_gpu_count() -> int:
+    if not os.path.isdir("/dev/"):
```

Comment:
If there is no dev dir could also return 0, though I don't think this is suppose to be able to run without having /dev/nvidia* paths